profiles/common/modules/features/nexteuropa_newsroom/theme/newsroom-service-items.tpl.php

```diff
@@ -0,0 +1,41 @@
+<?php
+/**
+ * @file
+ * Service items template.
+ */
+?>
+<?php if (count($items) > 0) : ?>
```

Comment:
Please clean up this file, replace all of the direct function and property calls to a template/preprocess function and print only variables here without creating them.